Content.Server/Interfaces/GameObjects/IOnDamageBehavior.cs

```diff
@@ -15,13 +16,20 @@ interface IOnDamageBehavior
         /// Gets a list of all DamageThresholds this component/entity are interested in.
         /// </summary>
         /// <returns>List of DamageThresholds to be added to DamageableComponent for watching.</returns>
-        List<DamageThreshold> GetAllDamageThresholds();
+        List<DamageThreshold> GetAllDamageThresholds() => null;
```

Comment:
why you add default implementations?

Reply:
(As a heads up, you can now do a multiline review on github. No need to open three reviews for the same thing!)
Without default implementations, components implementing IOnDamageBehavior would be forced to implement both OnDamageThresholdPassed (which requires GetAllDamageThresholds) and OnDamaged, when it might only need one of the two. Besides, since I extended an already existing interface, I felt like default implementations would be best as to prevent having to change all the components that already implement this interface.

Reply:
sure, sounds logical